config/environments/production.rb

```diff
@@ -70,6 +71,12 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
```

Comment:
this is the default and doesn't need to be set.

Reply:
Removed.